ui/src/app/sensors/components/sensor-logs-viewer.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+import {Observable} from 'rxjs';
+import {Sensor} from '../../../models';
+import {services} from '../../shared/services';
+import {FullHeightLogsViewer} from '../../workflows/components/workflow-logs-viewer/full-height-logs-viewer';
+
+function identity<T>(value: T) {
+    return () => value;
+}
+
+export const SensorLogsViewer = ({
+    namespace,
+    selectedTrigger,
+    sensor,
+    onClick
+}: {
+    namespace: string;
+    selectedTrigger: string;
+    sensor: Sensor;
+    onClick: (selectedNdoe: string) => void;
+}) => {
+    const [error, setError] = useState<Error>();
```

Comment:
```
ERROR in /Users/acollins8/go/src/github.com/argoproj/argo/ui/src/app/sensors/components/sensor-logs-viewer.tsx
./src/app/sensors/components/sensor-logs-viewer.tsx
[tsl] ERROR in /Users/acollins8/go/src/github.com/argoproj/argo/ui/src/app/sensors/components/sensor-logs-viewer.tsx(23,12)
      TS6133: 'error' is declared but its value is never read.

```

Reply:
you need an <ErrorNotice/> somewhere I think

Reply:
done.